prow/io/opener.go

```diff
@@ -480,3 +484,43 @@ func (o *opener) Iterator(ctx context.Context, prefix, delimiter string) (Object
 		}),
 	}, nil
 }
+
+func WriteContent(ctx context.Context, logger *logrus.Entry, opener Opener, path string, content []byte, opts ...WriterOptions) error {
```

Comment:
Why doesn't this function have `(o *opener)` as its receiver like the other functions in this file? If you have a good reason, can you please add it as a comment here?

Reply:
this function was copied from an old code. To my best guess this is to avoid adding one more function to the interface